src/components/TodoList/TodoList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
+type Props = {
+  todos: Todo[];
+  selectedTodo: Todo | null;
+  onShowTodo: (todo: Todo | null) => () => void;
```

Comment:
It will be better to create separete handler inside TodoList component
```suggestion
  onShowTodo: (todo: Todo | null) => void;
```